mod_test/controllers.py

```diff
@@ -233,6 +234,38 @@ def by_commit(commit_hash):
     return get_data_for_test(test, 'commit {commit}'.format(commit=commit_hash))
 
 
+@mod_test.route('/master/windows')
+@template_renderer('test/by_id.html')
+def latest_windows():
+    # Look up the hash
+    gh = GitHub(access_token=g.github['bot_token'])
+    repository = gh.repos(g.github['repository_owner'])(g.github['repository'])
+    latest_commit = repository.commits.master.get()
+    commit_hash = latest_commit['sha']
```

Comment:
Why would you fetch this if we store this field already? Take a look at the home controller for that ;)